src/time_tracker/invoices/handlers/spec.clj

```diff
@@ -45,6 +45,5 @@
 (s/def ::timers (timers-spec/normalized-timers-spec 0))
 
 (s/def ::invoice-data
-  (s/and (s/merge ::generate-invoice-params-coerced
-                  (s/keys :req-un [::users-spec/users ::timers-spec/timers]))
-         invoice-data-pred))
```

Comment:
@trycatcher yes I'm aware of why they don't match anymore. I'm saying that it should be modified to check for the new set of users.